### PR TITLE
Removed dependency to System.Reflection.Emit.Lightweight.

### LINKED ISSUE #318: System.Reflection.Emit.Lightweight Makes UWP Grumpy
Of course, UWP is a big grump all around. 😉

https://twitter.com/SuperJMN/status/1199987218733060096

### PR DESCRIPTION
**Pull request by Mike-EEE**
Fixes #318

### CODE REVIEW

**Comment by Mike-EEE** [comment]
Hey @SuperJMN let's continue the conversation here. If you can post code please provide it here.  I am installing the internet now with UWP SDK. :P

**Comment by Mike-EEE** [comment]
OK @SuperJMN... I have cloned your repo from https://github.com/SuperJMN/SuppaDesigner/tree/exs ... I updated the dependency on `Designer.Core` to `2.1.9.607-llceyabv` and do not see the error that you are experiencing.  I have load/saved (pretty cool!) a project file, but nothing results in an error.  Do you have steps to reproduce?

**Comment by Mike-EEE** [comment]
FWIW, I am using `Developer Mode` in Settings.  Not sure if that is the difference:

File: src/ExtendedXmlSerializer/ExtendedXmlSerializer.csproj [hunk 1]
```diff
@@ -56,7 +56,7 @@
     <PackageReference Include="NReco.LambdaParser" Version="1.0.10" />
     <PackageReference Include="Sprache" Version="2.2.0" />
     <PackageReference Include="System.Interactive" Version="4.0.0" />
-    <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.6.0" />
     <PackageReference Include="System.Collections.Immutable" Version="1.6.0" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.2" />
```
Comment (MagicAndre1981) [inline src/ExtendedXmlSerializer/ExtendedXmlSerializer.csproj]: why not use[ version 4.6](https://www.nuget.org/packages/System.Runtime.CompilerServices.Unsafe/4.6.0)?

Comment (Mike-EEE) [reply src/ExtendedXmlSerializer/ExtendedXmlSerializer.csproj]: Yeah good point, @MagicAndre1981... I somehow got an out-of-date version.  Our `Microsoft.SourceLink.GitHub` was also out of date, so went ahead and did that as well.

**Pull request merged**